views/js/controller/inspectResults.js

```diff
@@ -105,6 +106,24 @@ define([
                     taskCreationData : {uri : uniqueValue}
                 }).start();
             });
+
+            binder.register('open_url', function(actionContext) {
+                const data = _.pick(actionContext, ['uri', 'classUri', 'id']);
+
+                request(this.url, data,'POST')
```

Comment:
Should you `POST` or `GET`? Because the backend is not checking that and returns what you want with the default `GET`.

Reply:
It needs to be POST